dbt/models/sale/docs.md

```diff
@@ -0,0 +1,3 @@
+{% docs vw_ias_salesval_upload.sql %}
```

Comment:
**[Nitpick, non-blocking]** I missed this on the the first go-round, but to make this match our existing docs we should remove the `.sql` prefix! Here's an example of another docs block:

https://github.com/ccao-data/data-architecture/blob/ba9eadfed62f62f24dcbe73b8ccb903a14d6bae1/dbt/models/default/docs.md?plain=1#L40

Not entirely necessary, but maybe good for consistency?

```suggestion
{% docs vw_ias_salesval_upload %}
```